src/command/feed/print.ts

```diff
@@ -26,26 +27,39 @@ export class Print extends FeedCommand implements LeafCommand {
     await super.init()
 
     const topic = this.topic || this.bee.makeFeedTopic(this.topicString)
-    this.console.info('Looking up feed topic ' + topic + '...')
-    const addressString = this.address || (await this.getAddressString())
-    const reader = this.bee.makeFeedReader('sequence', topic, addressString)
-    const { reference, feedIndex, feedIndexNext } = await reader.download()
+    const spinner = createSpinner('Looking up feed topic ' + topic)
+    spinner.start()
+    try {
+      const addressString = this.address || (await this.getAddressString())
+      const reader = this.bee.makeFeedReader('sequence', topic, addressString)
+      const { reference, feedIndex, feedIndexNext } = await reader.download()
+
+      if (!this.stamp) {
+        this.stamp = await pickStamp(this.beeDebug, this.console)
```

Comment:
`spinner` needs to be stopped before this and then resumed, otherwise both the spinner and the stamp picker are active and it is glitchy